controllers/volsync/vshandler.go

```diff
@@ -2110,3 +2105,46 @@ func (v *VSHandler) checkLastSnapshotSyncStatus(lrs *volsyncv1alpha1.Replication
 
 	return !completed
 }
+
+func (v *VSHandler) DisownVolSyncManagedPVC(pvc *corev1.PersistentVolumeClaim) error {
+	// TODO: Remove just the VRG ownerReference instead of blindly removing all ownerreferences.
+	// For now, this is fine, given that the VRG is the sole owner of the PVC after DR is enabled.
+	pvc.ObjectMeta.OwnerReferences = nil
+	v.deleteAnnotation(pvc, ACMAppSubDoNotDeleteAnnotation)
+
+	return v.client.Update(v.ctx, pvc)
+}
+
+func (v *VSHandler) deleteAnnotation(obj client.Object, annotationName string) {
+	annotations := obj.GetAnnotations()
+	delete(annotations, annotationName)
+	obj.SetAnnotations(annotations)
```

Comment:
Can we replace this with this?

    delete(obj.Annotations, annotationName)

Reply:
We can.

Reply:
done